src/styles.scss

```diff
@@ -1,13 +1,37 @@
 @import 'ag-grid-community/styles/ag-grid.css';
 @import 'ag-grid-community/styles/ag-theme-quartz.css';
-@import 'ag-grid-community/styles/ag-theme-alpine.css';
+@import '@angular/material/theming';
 
 html, body {
-  //height: 100%;
+}
+
+.ag-theme-quartz {
+  --ag-background-color: #F7F7F7;
+  --ag-odd-row-background-color: #EBF2F3;
+  --ag-row-border-style: none;
+  --ag-font-family: Inter, "Helvetica Neue", sans-serif;
+}
+
+html {
+  --mat-sidenav-container-background-color: #F7F7F7;
+}
+.mat-accent {
+  --mat-option-selected-state-label-text-color: green;
+}
+
+//.mat-mdc-mini-fab {
+  //--mdc-fab-container-color: black;
+  //--mat-mdc-fab-color: #E84E0F;
+  //--mdc-fab-container-color: green;
+  //--mat-mdc-fab-color: green;
+  //
+  //--mat-mdc-fab-color: green;
+//}
+
+.ag-theme-quartz .ag-header-cell-text {
+  text-transform: Uppercase;
 }
```

Comment:
Include the following to make the table neater: 
`.ag-root-wrapper.ag-layout-normal { border-radius: 0;}`